src/industry_cmd.cpp

```diff
@@ -1126,9 +1126,15 @@ static void ProduceIndustryGoods(Industry *i)
 		uint32 r;
 		uint num;
 		if (Chance16R(1, 14, r) && (num = indsp->number_of_sounds) != 0 && _settings_client.sound.ambient) {
-			SndPlayTileFx(
-				(SoundFx)(indsp->random_sounds[((r >> 16) * num) >> 16]),
-				i->location.tile);
+			for (size_t j = 0; j < lengthof(i->last_month_production); j++) {
+				if (i->last_month_production[j] > 0) {
+					// Play sound since last month had production
```

Comment:
Coding style for single line comments (see [Coding style](https://wiki.openttd.org/Coding_style#Other_important_rules))
```suggestion
					/* Play sound since last month had production */
```